project/src/UserModel.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  age: {
+    type: Number,
+  },
+  jobTitle: {
+    type: String,
+  }
+});
+
+const User = mongoose.model('users', userSchema);
```

Comment:
inside the model function you don't need to pass the plural form of the object. For example you could have passed `user` instead of `users`